### ISSUE TITLE
[update] npm update to 2.9.0 fails and lacks documentation

### ISSUE THREAD

**Issue by MacLake** [fomantic/Fomantic-UI#2549]
# Bug Report
[The installation documentation](https://fomantic-ui.com/introduction/getting-started.html#installing-via-npm) mentions that from 2.9.0 on you need to manually run `npx gulp install` in `node_modules/fomantic-ui`. But also when updating to 2.9.0 the code in the `src` folder of your project needs to be updated.

## Steps to reproduce

1. Make sure you have a setup that runs `npx gulp build`
2. `npm update` in the `semantic` folder.
3. a) When running  `npx gulp install` in `semantic/node_modules/fomantic-ui` like described for the initial installation, or
    b)  in `semantic/` this leads to an error:
```zsh
Installing
------------------------------
Site folder exists, merging files (no overwrite) src/site
[02:20:57] 'create install files' errored after 2.24 ms
[02:20:57] Error: ENOENT: no such file or directory, stat './src/_site'
    at Object.statSync (node:fs:1590:3)
    at exports.copyDirSyncRecursive (/home/jens/develop/fomantic-ui/semantic/node_modules/wrench-sui/lib/wrench.js:244:23)
    at /home/jens/develop/fomantic-ui/semantic/tasks/install.js:339:12
    at taskWrapper (/home/jens/develop/fomantic-ui/semantic/node_modules/undertaker/lib/set-task.js:13:15)
    at bound (node:domain:433:15)
    at runBound (node:domain:444:12)
    at asyncRunner (/home/jens/develop/fomantic-ui/semantic/node_modules/async-done/index.js:55:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
[02:20:57] 'install' errored after 41 s
```

## Expected documentation and result
The documentation should mention how to run the install script for update, especially in which directory it should be run.

The default theme in `src/`should be updated to version 2.9.0, resulting in css and ja files with `2.9.0` in the header.

## Actual result
Update in `src` is incomplete.

## Version
2.9.0

[edit: path in step 3. b) corrected]

**Comment by lubber-de** [comment]
This was possibly already ajusted by https://github.com/fomantic/Fomantic-UI-Docs/pull/392

**Comment by MacLake** [comment]
@lubber-de Okay, thanks for pointing to the changed docs. Anyway, what I mean is that not only in the installation, but also in the update the installation script needs to run, which currently is not stated in the docs.

And the other thing, maybe I should write another issue for it: What about the failure when running `npx gulp install` for updating? Since I was not sure if it is a bug or it was just not clear for me which steps exactly are necessary for updating, I just wrote a single issue.

**Comment by lubber-de** [comment]
I cannot reproduce this unfortunately. After running npm update and then "npx gulp install" inside the fomantic-ui folder, the source files are updated
![image](https://user-images.githubusercontent.com/18379884/202653963-d5f5e90e-8082-468a-a216-8c5e3cca82a6.png)
After that running `npx gulp build` inside semantic folder creates new semantic.js/css files containing 2.9.0 inside the header

**Comment by MacLake** [comment]
In the meantime I have solved this issue for myself by using a fresh install and adding my self-written files to the project and applying patches to my themes. As I have a project which relies heavily on accordions, I had to use a 2.9.0 beta version because of the accordion issue with new Safari versions. Maybe that triggered the issue.

So if nobody else cares, I wouldn’t mind if you close this issue.
